spiketoolkit/validation/qualitymetrics.py

```diff
@@ -2,7 +2,8 @@
 import spiketoolkit as st
 
 
-def compute_unit_SNR(recording, sorting, unit_ids=None, save_as_property=True):
+def compute_unit_SNR(recording, sorting, unit_ids=None, save_as_property=True, mode='mad',
+                     seconds=10, max_num_waveforms=1000, filter=False, freq_min=300, freq_max=6000):
```

Comment:
To be sure that function is correctly used I would put filter=True.

So if you known what you do you can always disable but if you don't known it is better to be safe.

Reply:
No sorry i misunderstood.
I would put **already_filtered=False** or at least **fileterd=False**
filter is ambiguous because it can both mean already filtered or apply filter

Reply:
Mmm, I think it's better to filter outside. Extracting waveforms/templates on a FilterRecordingExtractor is very bad, because it's lazy and it filters all snippets.

I agree, but first we should add a `cache` arg to the bandpass_filter. If operations like get_waveforms etc. are used the `cache=True` should be used so a big filter is done once only?

Agree? I can make it today

Reply:
done.

I think `filter` is clear enough with the docs, personally. I would leave it false by default (by the time you run spike sorting you probably have filtered your data already...).
Or we can call it apply_filter :)

Reply:
used `apply_filter`